drivers/hmis/app/graphql/types/hmis_schema/has_assessments.rb

```diff
@@ -12,13 +12,14 @@ module HasAssessments
       extend ActiveSupport::Concern
 
       class_methods do
-        def assessments_field(name = :assessments, description = nil, **override_options, &block)
+        def assessments_field(name = :assessments, description = nil, filter_type_name: nil, filter_omit: [], **override_options, &block)
           default_field_options = { type: HmisSchema::Assessment.page_type, null: false, description: description }
           field_options = default_field_options.merge(override_options)
           field(name, **field_options) do
             argument :sort_order, Types::HmisSchema::AssessmentSortOption, required: false
             argument :roles, [Types::Forms::Enums::FormRole], required: false
             argument :in_progress, GraphQL::Types::Boolean, required: false
+            filters_argument HmisSchema::Assessment, filter_type_name: filter_type_name, filter_omit: filter_omit
```

Comment:
This doesn't allow for passing any kwargs to the filter argument (even though it accepts `**kwargs`).
Can we do something like this instead, for more flexibility?
```
def assessments_field(..., filter_args: {})

filters_argument HmisSchema::Assessment, **filter_args

``